content/country/switzerland/index.de.md

```diff
@@ -0,0 +1,76 @@
+{{% /expander %}}
+
+
+### Deutschland
+
+Von Deutschland aus gibt es mehrere grenzüberschreitende Verbindungen in die Schweiz. Die meistgenutzte ist hier aus Freibug im Breisgau kommend nach Basel. Hier verkehren ICE-Züge tagsüber meist ein bis zweimal pro Stunde pro Richtung. Diese kommen entweder aus Richtung Hamburg, Kassel, Frankfurt (Main) oder Hannover, Dortmund, Köln, Frankfurt (Main) und fahren dann zwischen Mannheim und Basel auf der gleichen Strecke. In der Schweiz werden einige fortgeführt bis nach Zürich, Interlaken, Bern oder Chur. Ebenfalls verkehren vereinzelt EC- und ECE-Züge auf der Strecke, die ebenfalls aus Richtung Mannheim kommen. Auch Regionalzüge verkehren aus Deutschland bis Basel. Zu beachten ist, dass bis zum Bahnhof Basel Bad Bf (Badischer Bahnhof) auch FIP-Freifahrtsscheine der DB noch gültig sind, da dies ein offizieller Grenzpunkt der DB und SBB ist. Weitere wichtige grenzüberschreitende Verbindungen sind die Strecken Stuttgart - Zürich über Schaffhausen (Grenzpunkt, DB und SBB Freifahrtsscheine gültig) sowie München - Zürich über Lindau und St. Gallen, auf denen IC- bzw. ECE-Züge verkehren. Dazu kommen noch einige grenzüberschreitende Verbindungen im Regionalverkehr. Auch mit Fähren, die teilweise mit FIP nutzbar sind, kann die Schweiz erreicht werden.  
+
+### Frankreich
+
+Von Frankreich aus sind sowohl Verbindungen mit dem französischen Hochgeschwindigkeitszug TGV als auch Regionalverbindungen möglich. Von Paris aus kann mit dem TGV sowohl Basel (teilweise auch direkt weiter nach Zürich) als auch Genf erreicht werden. In diesen Zügen gelten jedoch auf grenzüberschreitenden Relationen keine normalen FIP-Tickets, sondern es müssen FIP-Globalpreise gekauft werden. Bei den Regionalzügen, die über diverse Grenzübergänge verkehren, reicht ein FIP 50 Ticket oder FIP-Freifahrttscheine beider betreibender Bahngesellschaften.  
+
+### Italien
+
+Aus Italien kann sowohl mit grenzüberschreitenden Fern- als auch Regionalzügen in die Schweiz gereist werden. Von Mailand kann sowohl über Como als auch über Domodossola das Schweizer Netz erreicht werden. Die Züge über Como fahren meist weiter in Richtung Zürich, die über Domodossola nach Bern und Basel. Auf beiden Routen verkehren sowohl EuroCity-Verbindungen als auch Regionalzüge. Andere Grenzübergänge können nur mit Regionalzügen überquert werden. Bei den grenzüberschreitenden EuroCity-Zügen ist eine Reservierung nötig, die einen Aufschlag beinhaltet. Diese kann vergleichsweise ziemlich teuer sein.   
+{{% highlight tip %}}
+Bei der Verbindung Mailand - Como - Schweiz kann es sich sehr lohnen zwischen Mailand und Chiasso (erster Bahnhof in der Schweiz) mit dem Regionalverkehr zu fahren. Auf der Strecke verkehrt Trenord, die keine FIP-Tickets anerkennen. Jedoch ist das Ticket vergleichsweise günstig, im Gegensatz zu der benötigten grenzüberschreitenden Reservierung in den EuroCity-Zügen. Ein Umstieg in Chiasso kann daher eine sehr viel kostengünstigere Option sein.
```

Comment:
For the Frankfurt-Milano EC(E) the surcharge is 11€ in 2nd and 13€ in 1st class which includes a reservation for the train all the way to Frankfurt. Given that you save some time and avoid the risk of missing your connecting train, I think this is not too bad.

Reply:
Thanks for the input. I will change the description so the user has different choices. A cheap and and a comfortable one :)